tests/js/client/shell/shell-admin-cluster-rebalance.js

```diff
@@ -144,5 +175,192 @@ function clusterRebalanceSuite() {
   };
 }
 
+function clusterRebalanceOtherOptionsSuite() {
+  return {
+
+    setUpAll: function () {
+      prevDB = db._name();
+      db._createDatabase(database);
+      db._useDatabase(database);
+      for (let i = 0; i < 3; ++i) {
+        db._create("col" + i, {numberOfShards: 2, replicationFactor: 2});
+      }
+      for (let i = 0; i < 3; ++i) {
+        for (let j = 0; j < 1000; ++j) {
+          db["col" + i].insert({"value": j});
+        }
+      }
+    },
+
+    tearDownAll: function () {
+      for (let i = 0; i < 3; i++) {
+        db._drop("col" + i);
+      }
+      db._useDatabase(prevDB);
+      db._dropDatabase(database);
+    },
+
+    testCalcRebalanceStopServer: function () {
+
+      let dbServers = instanceManager.arangods.filter(arangod => arangod.instanceRole === "dbserver");
+      try {
+        for (let i = 0; i < dbServers.length; ++i) {
+          dbServer = dbServers[i];
+          assertTrue(suspendExternal(dbServer.pid));
+          let serverHealth = null;
+          let startTime = Date.now();
+          let result = null;
+          do {
+            wait(2);
+            result = getServersHealth();
+            assertNotEqual(result, null);
+            serverHealth = result[dbServer.id].Status;
+            assertNotEqual(serverHealth, null);
+            const timeElapsed = (Date.now() - startTime) / 1000;
+            const seconds = Math.round(timeElapsed);
+            if (seconds > 300) {
+              throw new Error("Server expected status not acquired");
+            }
+          } while (serverHealth !== "FAILED");
+          dbServer.suspended = true;
+          const serverShortName = result[dbServer.id].ShortName;
+          assertNotEqual(serverHealth, "GOOD");
+          startTime = Date.now();
+          let serverUsed;
+          do {
+            wait(2);
+            serverUsed = false;
+            for (let j = 0; j < 3; ++j) {
+              result = arango.GET("/_admin/cluster/shardDistribution").results["col" + j].Plan;
+              for (let key of Object.keys(result)) {
+                if (result[key].leader === serverShortName) {
+                  serverUsed = true;
+                  break;
+                }
+                result[key].followers.forEach(follower => {
+                  if (follower === serverShortName) {
+                    serverUsed = true;
+                    return;
+                  }
+                });
+              }
+              if (serverUsed === true) {
+                break;
+              }
+              const timeElapsed = (Date.now() - startTime) / 1000;
+              const seconds = Math.round(timeElapsed);
+              if (seconds > 300) {
+                throw new Error("Moving shards from server in ill state not acquired");
+              }
+            }
+          } while (serverUsed);
+
+          result = getRebalancePlan(true, true, true);
+          let moves = result.result.moves;
+          if (moves.length > 0) {
```

Comment:
are moves expected here? Right now, there can be 0 moves and the test will pass :thinking: